test/topology_custom/test_tablets.py

```diff
@@ -382,6 +382,104 @@ async def test_read_of_pending_replica_during_migration(manager: ManagerClient,
+
+    # Open the logs of both servers.
+    log_tasks = [asyncio.create_task(manager.server_open_log(srv.server_id)) for srv in servers]
+
+    # Collect results (completion order doesn't matter).
+    (src_node_id, src_shard) = await source_task
+    node_ids = [await task for task in node_id_tasks]
+    crd_id = await crd_task
+    logs = [await task for task in log_tasks]
+
+    # The destination node is the node that is not the source node. We always use shard#0 on the destination.
+    src = node_ids.index(src_node_id)
+    dst = 1 - src
+    dst_shard = 0
+
+    # The coordinator node is one of the two nodes. The non-coordinator node is the other node.
+    crd = node_ids.index(crd_id)
+    ncr = 1 - crd
+
+    # Four variations are possible:
+    #
+    # source  destination  coordinator  non-coordinator
+    # ------  -----------  -----------  ---------------
+    # node#0       node#1       node#0           node#1
+    # node#0       node#1       node#1           node#0
+    # node#1       node#0       node#0           node#1
+    # node#1       node#0       node#1           node#0
+    logger.info(f"src id={servers[src].server_id} ip={servers[src].ip_addr} node={node_ids[src]} shard={src_shard}")
+    logger.info(f"dst id={servers[dst].server_id} ip={servers[dst].ip_addr} node={node_ids[dst]} shard={dst_shard}")
+    logger.info(f"crd id={servers[crd].server_id} ip={servers[crd].ip_addr} node={node_ids[crd]}")
+    logger.info(f"ncr id={servers[ncr].server_id} ip={servers[ncr].ip_addr} node={node_ids[ncr]}")
+
+    logger.info("Decommissioning src")
+    # Inject a wait-for-message into the topology coordinator. We're going to block decommission right after entering the "tablet draining" transition state.
+    await manager.api.enable_injection(servers[crd].ip_addr, "suspend_decommission", one_shot=True)
+
+    # Initiate decommissioning the source node, and wait until the coordinator reaches "tablet draining".
+    crd_log_mark = await logs[crd].mark()
+    decomm_task = asyncio.create_task(manager.decommission_node(servers[src].server_id))
+    await logs[crd].wait_for('entered `tablet draining` transition state', from_mark=crd_log_mark)
+
+    logger.info("Moving tablet from src to dst")
+    # Move the tablet from the source node to the destination node. Ask the non-coordinator node to do it, as the coordinator node is suspended. Wait until the
+    # storage service on the non-coordinator node confirms it has seen the topology state machine as busy, and that it has kept the transition state intact.
+    ncr_log_mark = await logs[ncr].mark()
+    move_task = asyncio.create_task(manager.api.move_tablet(servers[ncr].ip_addr, "test", "tabmv_decomm",
+                                                            node_ids[src], src_shard, node_ids[dst], dst_shard, token))
+    await logs[ncr].wait_for('transit_tablet\([^)]+\): topology busy, keeping transition state', from_mark=ncr_log_mark)
```

Comment:
please note, `\(` is an invalid escape sequence. Python interpreter complains at seeing it, but it does not choke on it, instead if just falls back to literal `\(` (two characters). probably we could instead use the raw string like:
```py
await logs[ncr].wait_for(r'transit_tablet\([^)]+\): topology busy, keeping transition state', from_mark=ncr_log_mark)
```

so that Python does not try to interpret `\(` as an escape sequence, and considers "\" as literal characters?